os/net/mac/csma/csma-security.c

```diff
@@ -238,8 +238,14 @@ csma_security_parse_frame(void)
   }
 
   if(packetbuf_attr(PACKETBUF_ATTR_SECURITY_LEVEL) == 0) {
+    /* From https://github.com/contiki-ng/contiki-ng/issues/1610 */
+    /* This should reject unencrypted packets */
+    #ifdef LLSEC802154_REJECT_INSECURE
```

Comment:
Thanks for the contribution. Perhaps this should be named CSMA_CONF_LLSEC_REJECT_INSECURE, however, and be put in csma-security.h among the other configuration parameters for the CSMA implementation?

Reply:
I agree, but that could be a different PR? This one seems fairly critical to get in, and the define naming is just a cosmetic thing with lower priority IMO.

Reply:
Yes, I think that's fine.